src/ast/isReference.js

```diff
@@ -4,6 +4,8 @@ export default function isReference ( node, parent ) {
 	}
 
 	if ( node.type === 'Identifier' ) {
+		if ( !parent ) return true;
```

Comment:
It's not clear from the commit code alone why there'd be no parent. Is `parent` not the node that most immediately contains `node`? If it is, then only `Program` nodes don't have parents, right?